components/home/mcp-card.tsx

```
@@ -0,0 +1,185 @@
+            }
+            await navigator.clipboard.writeText(toCopy)
+            setCopied(true)
+            setTimeout(() => setCopied(false), 2000)
+        } catch (e) {
+            console.error("Error copying to clipboard", e)
+            toast.error("Error copying to clipboard")
+        }
+    }
+
+    // Show the merged button if there is a sensitive envVar or API key field
+    const hasSensitive = (server.envVars && server.envVars.some((env) => env.sensitive || env.label === 'x-meeting-baas-api-key' || env.label === 'x-api-key' || env.label === 'API_KEY')) || server.localConfig
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)